readme.md

```
@@ -21,15 +21,15 @@ Please note that [node.js](http://nodejs.org/) and npm is required. Specifically
 1. Fork this repo (see [GitHub help](https://help.github.com/articles/fork-a-repo/) for details)
 1. Clone your fork: `git clone https://github.com/YOUR-USERNAME/couchdb-fauxton.git`
 1. Go to your cloned copy: `cd couchdb-fauxton`
-1. Set up the upstream repo: 
+1. Set up the upstream repo:
     * `git remote add upstream https://github.com/apache/couchdb-fauxton.git`
     * `git fetch upstream`
     * `git branch --set-upstream-to=upstream/main main`
 1. Download all dependencies: `npm install`
 1. Make sure you have CouchDB installed.
     - Option 1 (**recommended**): Use `npm run docker:up` to start a Docker container running CouchDB with user `tester` and password `testerpass`.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)